packages/js/email-editor/package.json

```diff
@@ -0,0 +1,91 @@
+{
+	"name": "@mailpoet/email-editor",
+	"private": true,
+	"sideEffects": [
+		"build-style/**",
+		"src/**/*.scss"
+	],
+	"version": "0.1.0",
+	"scripts": {
```

Comment:
Are there any tests in this package?

Reply:
We have tests in the PHP package, and more will be added to the JS package in another PR.

Reply:
More tests will be added here https://github.com/orgs/woocommerce/projects/297/views/1?pane=issue&itemId=98391871